### DISCUSSION TITLE
Any UI library recommendation?

### DISCUSSION THREAD

**Discussion by borsemayur2** [crewdevio/Snel#25] [Q&A] [answered]
I'm looking forward to use `snel` for my upcoming book on Deno. I'm not versed with svelte and will learn it along. Please do recommend some UI libraries I can try with `snel`.

P.S. Should we enable `discussions` tab so these questions can be asked there?

**Comment by geoffreysmith** [comment, marked as answer]
IBM Carbon is really full featured but the Svelte/Carbon documentation and examples are not in alignment, they're very fragile (for example I had issues not building from the npx/degit template and instead by hand and found things just didn't work). Material UI is standard too, but given the fragility in how things like the rollup file changes without being documented and you pretty much need to use the template generation I'm concerned on how it'd work with Snel. I'm working on a POC of moving to Snel so we'll see how tied some of these things are the Svelte compiler (for lack of better term) or how much they are tied to node.js environment messiness.

**Comment by buttercubz** [comment]
> IBM Carbon is really full featured but the Svelte/Carbon documentation and examples are not in alignment, they're very fragile (for example I had issues not building from the npx/degit template and instead by hand and found things just didn't work). Material UI is standard too, but given the fragility in how things like the rollup file changes without being documented and you pretty much need to use the template generation I'm concerned on how it'd work with Snel. I'm working on a POC of moving to Snel so we'll see how tied some of these things are the Svelte compiler (for lack of better term) or how much they are tied to node.js environment messiness.

you can see https://github.com/crewdevio/Snel-carbon
